### ISSUE TITLE
`sonarjs/todo-tag` can not be disabled with `eslint-disable-next-line`

### ISSUE THREAD

**Issue by JounQin** [SonarSource/SonarJS#5185]
```ts
// eslint-disable-next-line sonarjs/todo-tag
'@typescript-eslint/naming-convention': 0, // TODO: find better config
```

```log
  ✘  sonarjs/todo-tag

     Complete the task associated to this "TODO" comment


     packages/eslint-config/overrides.js:156:41
     154 |     '@typescript-eslint/explicit-module-boundary-types': 0,
     155 |     '@typescript-eslint/member-ordering': 2,
   > 156 |     // eslint-disable-next-line sonarjs/todo-tag
         |                                         ^
     157 |     '@typescript-eslint/naming-convention': 0, // TODO: find better config
     158 |     '@typescript-eslint/no-empty-function': 2,
     159 |     '@typescript-eslint/no-extraneous-class': [

✘ 1 problem (1 error, 0 warnings)
```

**Comment by JounQin** [comment]
Workaround: `// eslint-disable-line sonarjs/todo-tag -- TODO: find better config`

**Comment by kebetsi** [comment]
We have stopped moderating GitHub issues. I will need to check why they are available again.

Could you please report this issue in [our community forum](https://community.sonarsource.com/) as [the README suggests?](https://github.com/SonarSource/SonarJS?tab=readme-ov-file#sonarsource-community-forum)

**Comment by JounQin** [comment]
@kebetsi https://community.sonarsource.com/t/sonarjs-todo-tag-can-not-be-disabled-with-eslint-disable-next-line/138183
